php/libraries/User.class.inc

```diff
@@ -66,26 +66,41 @@ class User extends UserPermissions
         $row = $DB->pselectRow($query, array('UID' => $username));
```

Comment:
Yeah good point. I think it's possible for that situation to arise in `request_account` which is exactly how I stumbled on this. I'll add another error message.